src/services/api/createAppApiClient.ts

```diff
@@ -126,3 +127,20 @@ const getTasks =
       return res.data
     } catch (err) {}
   }
+export type DeleteTaskByIdRequest = {
+  _id: string
+}
+
+export type DeleteTaskByIdResponse = {
+  success: boolean
+  data: Task
+}
+
+const DeleteTaskById =
+  (api: AxiosInstance) =>
+  async (data: DeleteTaskByIdRequest): Promise<DeleteTaskByIdResponse | undefined | null> => {
+    try {
+      const res = await api.delete<DeleteTaskByIdResponse>("/task/" + data._id)
```

Comment:
```suggestion
      const res = await api.delete<DeleteTaskByIdResponse>(`/task/${data._id}`)
```
2021 rồi đừng sống ở 2015 nữa e

Reply:
sory anh do trên trường làm đồ án java nên em quen